api/src/Entity/Activity.php

```diff
@@ -19,9 +20,12 @@
  * @ORM\Entity
  */
 #[ApiResource(
-    collectionOperations: ['get', 'post'],
-    itemOperations: [
+    collectionOperations: [
         'get',
+        'post' => ['denormalization_context' => ['groups' => ['write', 'create']]],
+    ],
+    itemOperations: [
+        'get' => ['normalization_context' => self::ITEM_NORMALIZATION_CONTEXT],
```

Comment:
Müsstest du diesen `normalization_context` jetzt nicht eigentlich auch bei `post` und `patch` hinschreiben? Ich dachte das machen wir standardmässig so, darum auch die Konstante, und weichen nur in Ausnahmefällen davon ab

Reply:
done